docs/sphinx/type-docs.rst

```diff
@@ -0,0 +1,4 @@
+====================
+Auto Docs by TypeDoc
+====================
+`TypeDocs <_static/typedocs/index.html>`_
```

Comment:
This linking is not working when running it on my side. It likely needs to be manually copied into a new directory within `docs/build/html`. I tested this by building using your `docs` Docker environment and opening `docs/build/html/index.html`.

Reply:
Fixed